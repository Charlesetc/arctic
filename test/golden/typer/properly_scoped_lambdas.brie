define main [
  define x < y = 3 >
  define l [
    x
  ]
  define x 2
  (l ()).y
]
