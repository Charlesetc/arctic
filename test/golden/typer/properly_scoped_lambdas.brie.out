root[(ident(define):: ident(main):: block[][(ident(define):: ident(x):: <y = (ident(3)::Integer)::Integer>::<y = Integer>)::, (ident(define):: ident(l):: block[][(ident(x)::)::]::Function)::, (ident(define):: ident(x):: ident(2)::Integer)::, ((ident(l)::Function ()::Unit)::<y = Integer>.y::Integer)::Integer]::Function)::]::